python/tests/test_ephemeris.py

```diff
@@ -186,3 +186,48 @@ def test_parameters():
   valid = 1
   fit_interval = 4
   assert not e.Ephemeris.is_params_valid(valid, fit_interval, toe, time)
+
+def test_glo_sat_state():
+  eph1 = e.Ephemeris(**{'sid': {'sat': 1, 'code': 3,},
+                     'toe': {'wn': 1884, 'tow': 260100.0,}, #from GPS
+                     'ura': 2.0,
+                     'fit_interval': 4 * 60 * 60,
+                     'valid': 1,
+                     'healthy': 1,
+                     'glo': {  #RINEX data at 2016-02-17T00:15:00
+                              'gamma' : 0,
```

Comment:
Hmm are there any satellites with non-zero gamma? Otherwise this wouldn't pick up an error in using gamma to compute the clock correction.